packages/dashboard/src/helpers/getFilteredData.ts

```diff
@@ -33,7 +33,7 @@ export const getFilteredData = (state: DashboardState, initialFilteredData = {},
 
         newFilteredData[index] = filterData(applicableFilters, formattedData)
       } else {
-        newFilteredData[index] = state.data[row.dataKey] || row.data
+        newFilteredData[index] = state.data[row.dataKey] || row.data || []

```

Comment:
nit: outside of scope, but this function has a repeatable pattern that could be abstracted.

Reply:
you mean like _data || []